javascript/packages/cli/src/cli.ts

```diff
@@ -87,21 +87,25 @@ const downloadBinaries = async (binaries: string[]): Promise<void> => {
           const progressBar = multibar.create(parseInt(contentLength, 10), 0);
           const reader = response.body?.getReader();
           const writer = fs.createWriteStream(path.resolve(name));
-
-          while (true) {
-            const read = await reader?.read()!;
-            if (read?.done) {
-              writer.close();
-              resolve();
-              break;
+          let i = true;
+          while (i) {
+            const read = await reader?.read();
+            if (read) {
+              if (read?.done) {
+                writer.close();
+                i = false;
+                resolve();
+              }
+
+              if (read?.value) {
+                loaded += read.value.length;
+                progressBar.increment();
+                progressBar.update(loaded, {
+                  filename: name,
+                });
+                writer.write(read.value);
+              }
             }
-
-            loaded += read.value.length;
-            progressBar.increment();
-            progressBar.update(loaded, {
-              filename: name,
-            });
-            writer.write(read.value);
           }
         }),
```

Comment:
`This part is not needed, having read?.done is enough`

Linters propose that this is needed. Did you try to remove it? Also I am not refactoring code (moving it around) in one PR. This can take place in different PR. The point of this PR is to introduce eslinter;